bundle/terranova/tnresources/app.go

```diff
@@ -23,6 +23,10 @@ func (*ResourceApp) PrepareConfig(input *resources.App) *apps.App {
 	return &input.App
 }
 
+func (r *ResourceApp) DoRefresh(ctx context.Context, id string) (*apps.App, error) {
```

Comment:
App also has `WaitAfterCreate` but with it's "no-refresh" variabnt, is it expected?